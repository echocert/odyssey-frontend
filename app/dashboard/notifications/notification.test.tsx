import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { create } from "react-test-renderer";
import Notifications from './page';
describe("Notifications Component", () => {
  it("renders the Notifications Component ", () => {
    render(<Notifications />);
  });

  it("Verify the snapshot of the Notifications Component", () => {
    const wrap = create(<Notifications />).toJSON();
    expect(wrap).toMatchSnapshot();
  });
});