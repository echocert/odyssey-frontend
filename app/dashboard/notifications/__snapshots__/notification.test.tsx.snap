// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications Component Verify the snapshot of the Notifications Component 1`] = `null`;
